### ISSUE TITLE
Selenium Library does not load for only *some* robot files

### ISSUE THREAD

**Issue by amankul** [nokia/RED#383]
Hi,

My setup is as follows -
1. Windows 10
2. Eclipse IDE Version: 2020-06 (4.16.0)
3. RED Robot Editor 0.9.5.202007240959

My project has 2 files at different locations.
AdminOppFields.robot – Does not give content assist upon hitting CTRL+SPACE
AdminOppFields-copy.robot – Does provide keywords in selenium lib.
Already tried content assist preferences, re-importing project, reloading library and stack overflow 😊
Any help would be really appreciated. 

![image](https://user-images.githubusercontent.com/43149375/89197534-7c339f00-d571-11ea-838c-00886ee9330c.png)

![image](https://user-images.githubusercontent.com/43149375/89197516-73db6400-d571-11ea-833e-934443a1c7cf.png)

**Comment by michalanglart** [comment]
Hi,

can you reproduce it with some minimal dummy project and provide it to us?

**Comment by amankul** [comment]
@michalanglart  Unfortunately doesn't reproduce when duped my project. As I said the behavior was inconsistent in my original project itself.  Anything else to watch out for?

**Comment by michalanglart** [comment]
To bo honest we've never encountered such situation and I have no clue why you would see it. Is your project located at single directory at you file system, or is it made out of couple using linking at eclipse level?

I assume both files have `Library    SeleniumLibrary`? import in settings section (I can see that for `AdminOppFields.robot`)?

**Comment by amankul** [comment]
Yes both files are exactly identical but in diff directories. My entire project is one single workspace dir but there're are diff directories beneath.

**Comment by michalanglart** [comment]
You could check if there is anything interesting in `metadata/.log` file - especially exceptions.

I understand that it does not work for this file regarding if you restart IDE or not?

This sound really weird..

**Comment by amankul** [comment]
[.log](https://github.com/nokia/RED/files/5037668/default.log)

Already did look here but did make any sense to me. Check it out.

**Comment by amankul** [comment]
@michalanglart Similar issue has been reported on Pycharm too. https://groups.google.com/forum/?utm_medium=email&utm_source=footer#!msg/robotframework-users/ZIx0TMLaBHk/O-X3vWAzCgAJ

**Comment by michalanglart** [comment]
RED has nothing to do with PyCharm. It's eclipse based plugin and its code is not used in any PyCharm/IntelliJ plugin. The .log file has nothing interesting regarding your case. I don't know how to help you unless we will be able to see it in our environment. Any chance to strip you project out of most of the files and just send us some minimal project on which you're reproducing that behaviour?

**Comment by amankul** [comment]
I know RED is independent of PyCharm. I was trying to check if the issue lies with Selenium lib since both RED and PyCharm use that. Have already tried to duplicate my employer project and kept only those and wasn't able to reproduce. Do you wanna do zoom sometime?
